packages/manager/modules/octavia-load-balancer/src/onboarding/controller.js

```diff
@@ -13,8 +27,24 @@ export default class OctaviaLoadBalancerOnboardingCtrl {
     }));
     this.illustration = illustration;
     this.cta = CTA[ovhSubsidiary] || CTA.DEFAULT;
+    this.privateNetworkHelpUrl =
+      PRIVATE_NETWORK_HELP[ovhSubsidiary] || PRIVATE_NETWORK_HELP.DEFAULT;
     this.atInternet = atInternet;
     this.ctaTrackName = `${TRACKING_CHAPTER_1}::${TRACKING_NAME}::add`;
+    this.$stateParams = $stateParams;
+    this.$http = $http;
```

Comment:
```suggestion
    this.$http = $http;
```

No need this if you move in routing see https://github.com/ovh/manager/pull/10001/files#r1329776682

Reply:
Done!